backend/models/issues.py

```diff
@@ -3,7 +3,7 @@
 
 import os
 
-from flask import current_app
+from flask import current_app, send_from_directory
```

Comment:
Is `send_from_directory` used anywhere here?

Reply:
no, i"ll fix it